test/e2e/specs/all-features-confirmation/record/presentation.spec.js

```diff
@@ -134,4 +140,29 @@ describe('View existing record,', function() {
             })
         });
     });
+    
+    describe("For multiple records fetched for particular filters", function() {
+        
+        beforeAll(function() {
+            var url = browser.params.url + "/record/#" + browser.params.catalogId + "/product-record:" + testParams.table_name +  "/luxurious=true";
+            browser.get(url);
+            chaisePage.waitForElement(element(by.css('.modal-dialog ')));
+        });
+        
+        it('A error modal window should appear with multiple records found error with correct title', function(){
+            var modalTitle = chaisePage.recordPage.getErrorModalTitle();
+            expect(modalTitle).toBe(testParams.multipleData.title, "The title of multiple record error pop is not correct");
+            
+        });
+        
+        it('On click of OK button the page should redirect to recordset page', function(){
+            chaisePage.recordPage.getErrorModalOkButton().then(function(btn){
```

Comment:
I don't know if we prefer `executeScript` or not. But you could rewrite it this way:
```javascript
chaisePage.recordPage.getErrorModalOkButton().then(function(btn){
    return btn.click();
}).then(function () {
    return browser.driver.getCurrentUrl();
}).then(function (currentUrl) {
    expect(currentUrl).toContain("recordset", "The redirection from record page to recordset in case of multiple records failed");
}).catch( function (err) {
    console.log(error);
}); 
```
@jrchudy Is the way this test case is written fine (using executeScript for getting current url)? or should we change it?